### DISCUSSION TITLE
Display pop up widget on symbol click

### DISCUSSION THREAD

**Discussion by blobglobs** [Imperial-lord/mapbox-flutter#5] [Q&A]
Hiya i hope you're good. 

I want to display a pop up widget when a symbol is clicked on mapbox. Not sure how to display a widget on top of the map after using onSymbolTapped. I've made another widget class by the way. Any help would be great, thanks a lot.

**Comment by Imperial-lord** [comment]
Do you wish to like show some information about the place or location you have tapped on? (Maybe title, description, distance, time etc.?)
